### DISCUSSION TITLE
Customising which fields are displayed in the CMS panel

### DISCUSSION THREAD

**Discussion by ldanielswakman** [tinacms/tinacms#3908] [Ideas]
In the Tina CMS panel, in order to ship this UI to clients, I would need to customise the list to fit the client's needs. For instance, in this example below I want to be able to define which fields are visible in the collection overview

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/4538367/236334497-99c42805-bef4-44af-b37c-edcd4316175b.png">

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/4538367/236334621-47abf887-ff3d-44a6-8207-4cdc8b42beb7.png">

**Comment by appinteractive** [comment]
Waiting for this too 

#3821
